src/data/faq_de.json

```diff
@@ -199,7 +199,12 @@
+                            "Die Einstellungen Ihres Smartphones zeigen die Begegnungsüberprüfungen der letzten 14 Tage an:",
+                            "<ul><li><b>iOS:</b> 'Einstellungen' -> 'Begegnungsmitteilungen' -> 'Status von Begegnungsaufzeichnungen' -> 'Begegnungsüberprüfungen'</li><li><b>Android:</b> 'Einstellungen' -> 'Google' -> 'COVID-19-Benachrichtigungen' -> 'Überprüfung auf mögliche Begegnungen'</li></ul>",
+                            "Wenn die App in den letzten 2 Wochen dauerhaft aktiv war, sollten Sie mindestens eine Überprüfung für jeden der letzten 14 Tage sehen.",
+                            "Bitte beachten Sie: Diese Übersicht zeigt nur die Historie an. Nach jeder Hintergrundaktualisierung werden die Begegnungen der letzten 14 Tage geprüft und das Risiko daraus ermittelt. Wichtig ist also, dass die Übersicht mindestens eine Begegnungsüberprüfung während der letzten 24 Stunden anzeigt.",
+                            "Es kann vorkommen, dass hier Einträge für weniger als 14 Tage angezeigt werden, obwohl die Risiko-Ermittlung an jedem der letzten 14 Tagen korrekt durchgeführt wurde. Dafür gibt es mehrere Gründe:",
+                            "<ol><li>Diese Übersicht hat ein festes Limit von 100 Einträgen auf iOS Geräten. Auf Android-Geräten beträgt das Limit 300 Einträge. Wird dieses Limit überschritten, werden nur die neuesten Einträge behalten. Sie sehen dann typischerweise die Einträge von 7 Tagen.</li><li>Wenn Sie die App erst seit Kurzem nutzen (zum Beispiel seit 5 Tagen), werden entsprechend nur Überprüfungen nach der Installation der App angezeigt - mindestens eine für jeden Tag, an dem die Corona-Warn-App lief.</li></ol>"
```

Comment:
"Diese Übersicht hat ein festes Limit von 100 Einträgen auf iOS Geräten. Auf Android-Geräten beträgt das Limit 300 Einträge. Wird dieses Limit überschritten, werden nur die neuesten Einträge behalten. **Sie sehen dann typischerweise die Einträge von 7 Tagen.**"

On Android it is not expected that the limit would be reached and typically you see entries for 16 days. I would just remove the sentence "Sie sehen dann typischerweise die Einträge von 7 Tagen."